app/controllers/case_contacts/form_controller.rb

```diff
@@ -139,14 +139,22 @@ def case_contact_params
   # Deletes the current associations (from the join table) only if the submitted form body has the parameters for
   # the contact_type ids.
   def remove_unwanted_contact_types
-    if params.dig(:case_contact, :case_contact_contact_type_attributes)
-      @case_contact.case_contact_contact_type.destroy_all
+    begin
+      if params.dig(:case_contact, :case_contact_contact_type_attributes) # TODO this sometimes raises errors in prod.
```

Comment:
https://app.bugsnag.com/ruby-for-good/casa/errors/66370462a19ffe00080cfd3c?filters[event.unhandled]=true&filters[event.since]=30d&filters[app.release_stage]=production&filters[error.status]=new
<img width="1654" alt="Screenshot 2024-05-05 at 12 11 01 AM" src="https://github.com/rubyforgood/casa/assets/578159/28c1c261-a9cd-4fc0-a259-12a00f04d20d">